src/__utils__/api.js

```diff
@@ -12,6 +12,13 @@ export const getArticles = () => {
         })
 }
 
+export const getArticlesByTopic = (slug) => {
+    const path = `/articles/?topic=${slug}`
```

Comment:
Rather than having a separate API call here, you could use axios params with the getArticles page, which would add the optional query of topic if one is passed in. This would then mean you could re-use your ArticlesListView component and wouldn't have the need for ArticlesByTopicView as a separate component which will be repeating a lot of code 👍